06-building-a-program-start-to-finish/writing-tests/output/inverted-index-test.js

```diff
@@ -0,0 +1,157 @@
+const books = require('../books.json');
+const adventureBooks = require('../adventure-books.json');
+const emptyArray = require('../emptyBook.json');
+const invalidContent = require('../invalid-content.json');
+const invalidFile = require('../invalid-file.json');
+const invalidKey = require('../invalid-key.json');
+
+const invertedIndex = new InvertedIndex();
+
+describe('Files to be indexed', () => {
```

Comment:
You test suites names are a little confusing. It's not clear which functions you're testing.